src/arithmetic/arithmetic_expression.c

```diff
@@ -1320,6 +1321,16 @@ SIValue AR_TIMESTAMP(SIValue *argv, int argc) {
 	return SI_LongVal(TemporalValue_NewTimestamp());
 }
 
+//==============================================================================
+//=== Scalar functions =========================================================
+//==============================================================================
+
+SIValue AR_RANDOMUUID(SIValue *argv, int argc) {
+	char *buf = rm_malloc(UUID4_LEN * sizeof(char));
+	uuid4_generate(buf);
```

Comment:
Is there a reliable `random_bytes` implementation somewhere in the codebase? I've found only several usages of `rand` function in the code. I can rewrite uuid4 generation to use `rand` function, and get rid of the `uuid4` library, but it seems like it may be a bit slower compared to filling the array with bytes. What do you think?